rebalancer.py

```diff
@@ -169,18 +173,40 @@ def auto_enable():
         apdays = int(LocalSettings.objects.filter(key='AR-APDays')[0].value)
     else:
         LocalSettings(key='AR-APDays', value='7').save()
-        apdays = 7        
+        apdays = 7
     if enabled == 1:
-        channels = Channels.objects.filter(is_active=True, is_open=True, private=False).annotate(outbound_percent=((Sum('local_balance')+Sum('pending_outbound'))*1000)/Sum('capacity')).annotate(inbound_percent=((Sum('remote_balance')+Sum('pending_inbound'))*1000)/Sum('capacity'))
+        channels = Channels.objects.filter(is_active=True, is_open=True, private=False)
         filter_day = datetime.now() - timedelta(days=apdays)
         forwards = Forwards.objects.filter(forward_date__gte=filter_day)
         for channel in channels:
-            outbound_percent = int(round(channel.outbound_percent/10, 0))
-            inbound_percent = int(round(channel.inbound_percent/10, 0))
-            routed_in_apday = forwards.filter(chan_id_in=channel.chan_id).count()
-            routed_out_apday = forwards.filter(chan_id_out=channel.chan_id).count()
-            iapD = 0 if routed_in_apday == 0 else int(forwards.filter(chan_id_in=channel.chan_id).aggregate(Sum('amt_in_msat'))['amt_in_msat__sum']/10000000)/100
-            oapD = 0 if routed_out_apday == 0 else int(forwards.filter(chan_id_out=channel.chan_id).aggregate(Sum('amt_out_msat'))['amt_out_msat__sum']/10000000)/100
+            peer_channels = Channels.objects.filter(remote_pubkey=channel.remote_pubkey)
```

Comment:
I would think we would still need to aggregate iapD and oapD to ensure decision is taken at peer instead of channel level. Otherwise you can end up with one channel enabled and other disabled.

Reply:
https://github.com/cryptosharks131/lndg/pull/94/commits/9cc8480cfe8992af4f6abe753f099ba8142a5d95